src/test/java/core/basesyntax/db/FruitDaoImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.db;
+
+import core.basesyntax.db.storage.Storage;
+import java.util.Map;
+import java.util.TreeMap;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitDaoImplTest {
+    public static final Map<String,Integer> EXPECTED_RESULT = new TreeMap<>();
+
+    @BeforeEach
+    void setUp() {
+        EXPECTED_RESULT.put("banana",20);
+        EXPECTED_RESULT.put("orange",4);
+        EXPECTED_RESULT.put("apple",15);
+        EXPECTED_RESULT.put("grapple",76);
+    }
+
+    @AfterEach
+    void reset() {
+        EXPECTED_RESULT.clear();
+        Storage.fruits.clear();
+    }
+
+    @Test
+    void addingNewFruit_OK() {
```

Comment:
The test method name `addingNewFruit_OK` does not follow the recommended naming convention. Consider using a more descriptive.
```suggestion
    void addNewFruit_shouldAddFruitToStorage() {
```